provider/Providers/ISchemaQueryProvider.cs

```diff
@@ -101,6 +101,8 @@ public ColumnDefinition(string name)
 
     public string DatabaseType { get; set; } = null!;
 
+    public string CSharpType { get; set; } = null!;
```

Comment:
I'm confused about this property. It is defined under the providers, but assigned from the excel tool.

Additionally, it would be good not to put language-specific properties in the providers, which is meant to be agnostic. Instead, I've added a `TypeDisplay` class in `Utilities`, which allows you to convert a .NET `Type` to a C# alias.

Reply:
Created separate POCO class and inherited ColumnDefinition and added a property for CSharpType.